llvm/test/ThinLTO/X86/workload.ll

```diff
@@ -0,0 +1,67 @@
+; RUN: mkdir -p %t_baseline
+; RUN: opt -module-summary %S/Inputs/workload1.ll -o %t_baseline/workload1.bc
```

Comment:
Recommend including all the inputs here and use split-file (https://llvm.org/docs/TestingGuide.html#extra-files). That would make it easier to follow what is going on, especially since there is no code in this file currently.

Reply:
Also, the test is a little confusing because some of the specified imports (e.g. m1_f1 lists m2_f1 and vice versa) are not actually (transitively) referenced from the importing module. I guess this is just for testing purposes, but it would be good to have a test that shows/tests the usage for a case where the imports would normally be expected.

Reply:
Right, my intention was to show that for a few functions, they'll get imported "regardless". m3 pulls from m1 "normally".